crates/sui-rosetta/src/unit_tests/operations_tests.rs

```diff
@@ -17,14 +17,13 @@ async fn test_operation_data_parsing() -> Result<(), anyhow::Error> {
 
     let sender = SuiAddress::random_for_testing_only();
 
-    let data = TransactionData::new_pay_sui_with_dummy_gas_price(
-        sender,
-        vec![gas],
-        vec![SuiAddress::random_for_testing_only()],
-        vec![10000],
-        gas,
-        1000,
-    );
+    // TODO figure out an analogous operation for programmable transactions
```

Comment:
What's wrong with this one?

Reply:
It is generating transaction metadata that is really specific for PaySui. I'm not sure how/if we want to replace this with programmable transactions

Reply:
Ah okay -- the reason I asked is that even before Programmable Transactions hit the scene, this test seemed quite specific (PaySui is special for lots of reasons) so, not sure you're making anything better or worse here.